£Lerror_reason¢DtypeEvalueY.HTTPConnectionPool(host='intranet.ics.uci.eduhttp', port=80): Max retries exceeded with url: //intranet.ics.uci.edu (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1088d5b50>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))Ihttp_code¢DtypeEvalueÛQdownload_complete¢DtypeEvalueı